components/Transactions.tsx

```diff
@@ -9,9 +9,11 @@ export default function Transactions() {
   const { chain } = useNetwork();
 
   useEffect(() => {
+    console.log({ chain });
```

Comment:
let's remove these instances of console.log

Reply:
a lol totally missed the draft 🤦

Reply:
I would avoid adding reviewers to the draft, unless you want comments :)